cli/main.rs

```diff
@@ -61,6 +61,8 @@ impl fmt::Display for StrBase {
             Base::Base32PadLower => "base32pad",
             Base::Base32PadUpper => "base32padupper",
             Base::Base32Z => "base32z",
+            Base::Base36Lower => "base32lower",
+            Base::Base36Upper => "base32upper",
```

Comment:
Base32 is a typo here! Please fix it. Also, the [standard](https://github.com/multiformats/multibase/blob/master/multibase.csv) uses base36 and base36upper and is not explicit about lower-case being lower.

Reply:
Good catch, thanks a lot for the thorough review!

> Also, the [standard](https://github.com/multiformats/multibase/blob/master/multibase.csv) uses base36 and base36upper and is not explicit about lower-case being lower.

Yes, though implicitly lowercase is meant. I think it's good to have the constants more explicit.